src/test/java/seedu/address/testutil/CompanyBuilder.java

```diff
@@ -20,19 +21,25 @@ public class CompanyBuilder {
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_EMAIL = "amy@gmail.com";
     public static final String DEFAULT_TAG = "SoftwareEngineer";
+    public static final String DEFAULT_STARTDATE = "2024-07-07";
+    public static final String DEFAULT_ENDDATE = "2024-08-08";
 
     private Name name;
     private Phone phone;
     private Email email;
     private Set<Tag> tags;
+    private Date startDate;
+    private Date endDate;
 
     /**
      * Creates a {@code CompanyBuilder} with the default details.
      */
     public CompanyBuilder() {
         name = new Name(DEFAULT_NAME);
-        phone = new Phone(DEFAULT_PHONE);
+        phone = new Phone("85355255");
```

Comment:
I think this could be left as DEFAULT_PHONE

Reply:
resolved